src/analysis.zig

```diff
@@ -1851,6 +1851,7 @@ fn addOutlineNodes(allocator: std.mem.Allocator, tree: Ast, child: Ast.Node.Inde
         .error_set_decl,
         => return,
         .container_decl,
+        .container_decl_trailing,
```

Comment:
debating whether I should replace else bellow with an explicit list of tags

Reply:
I don't think that's necessary here :)